content/components/selectpanel.mdx

```diff
@@ -0,0 +1,222 @@
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>End</Box>                                      | Move focus to last item in listbox.                                                                                         |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>PageDown</Box>                                 | Move focus to last item in listbox.                                                                                         |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>Esc</Box>                                      | Close dialog and clear any changed inputs.                                                                                  |
+
+### Multi-select
+
+| Key                                                                                                                                                           | Description                         |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------- |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>ArrowDown</Box> | Navigate through the listbox items. |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>Space</Box>     | Select an item in the listbox.      |
+
+### Single-select
+
+| Key                                                                                                                                                           | Description                                              |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| <Box as="kbd" backgroundColor="canvas.subtle" borderRadius={1} borderColor="border.default" borderWidth={1} borderStyle="solid" py={1} px={2}>ArrowDown</Box> | Navigate through listbox items. Selection follows focus. |
+
+#### Example behavior
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <CustomVideoPlayer
+      width="100%"
+      src="https://user-images.githubusercontent.com/40274682/228391029-44ab995b-7a04-4b85-825e-dc77f90838dd.mp4"
+    />
+    <Caption>
+      Keyboard interaction with a single-select Select panel demonstrating selection following focus with navigation via
+      the ArrowDown key.
+    </Caption>
+  </div>
+</Box>
```

Comment:
It would be nice to add a "Related links" section where we link to similar components and related UI pattern guidelines.

Here's what come to mind:
```
- [Data saving guidelines](/ui-patterns/saving)
- [ActionMenu](/components/ActionMenu)
- [Select](/components/Select)
```